chaos-days/blog/2023-12-18-Dynamic-Scaling-with-Dataloss/index.md

```diff
@@ -0,0 +1,514 @@
+---
+layout: posts
+title:  "Dynamic Scaling with Dataloss"
+date:   2023-12-18
+categories: 
+  - chaos_experiment 
+  - bpmn
+tags:
+  - availability
+authors: ole
+---
+
+# Chaos Day Summary
+
+We continue our expirements with dynamically scaling by adding or removing brokers to a running cluster by now also testing wether
+we survive dataloss during the process.
+
+**TL;DR;**
+While scaling was successful eventually, no progress was made while the coordinator was down.
+
+<!--truncate-->
+
+## Dataloss on the Coordinator
+
+Zeebe uses Broker 0 as the coordinator for changes to the cluster topology.
+While changes can only be initiated by the coordinator, losing the coordinator and it's data should not prevent the scaling operation from completing.
+
+### Expected
+
+We first request a scale up operation from the coordinator.
+Shortly after this has started, we trigger dataloss on the coordinator.
```

Comment:
🔧 This should moved up, and explain a bit more the set up and the idea behind this.